components/mpas-ocean/src/shared/mpas_ocn_vmix.F

```diff
@@ -1583,9 +1583,26 @@ subroutine ocn_vmix_implicit(dt, meshPool, statePool, forcingPool, scratchPool,
                !$acc enter data copyin(tracerGroupSurfaceFlux)
 #endif
 
-               call ocn_tracer_vmix_tend_implicit(dt, vertDiffTopOfCell, layerThickness, tracersGroup, &
+               if (trim(groupItr % memberName) /= 'activeTracers' .and. &
+                   config_cvmix_background_diffusion_passive_enable) then
+                      vertDiffPassiveTopOfCell = vertDiffTopOfCell - &
+                         config_cvmix_background_diffusion + config_cvmix_background_diffusion_passive
+#ifdef MPAS_OPENACC
+                      !$acc update host (vertDiffPassiveTopOfCell)
+#endif
+!maltrud debug
+ do k = 1, maxLevelCell(1)
+  write(*,*)' DEBUG01: ',k,vertDiffPassiveTopOfCell(k,1)
+ enddo
```

Comment:
I'm guessing you meant to remove these debug lines?

Reply:
@vanroekel , yeah, thanks for catching that.  will remove.